frontend/.template-lintrc.js

```diff
@@ -1,11 +1,98 @@
+    "no-attrs-in-components": "warn",
+    "no-autofocus-attribute": "warn",
+    "no-block-params-for-html-elements": "warn",
+    "no-capital-arguments": "warn",
+    "no-class-bindings": "warn",
+    "no-curly-component-invocation": "warn",
+    "no-debugger": "warn",
+    "no-duplicate-attributes": "warn",
+    "no-duplicate-id": "warn",
+    "no-duplicate-landmark-elements": "warn",
+    "no-empty-headings": "warn",
+    "no-extra-mut-helper-argument": "warn",
+    "no-forbidden-elements": "warn",
+    "no-heading-inside-button": "warn",
+    "no-html-comments": "warn",
+    "no-implicit-this": "warn",
+    "no-index-component-invocation": "warn",
+    "no-inline-styles": "warn",
+    "no-input-block": "warn",
+    "no-input-tagname": "warn",
+    "no-invalid-aria-attributes": "warn",
+    "no-invalid-interactive": "warn",
+    "no-invalid-link-text": "warn",
+    "no-invalid-link-title": "warn",
+    "no-invalid-meta": "warn",
+    "no-invalid-role": "warn",
+    "no-link-to-positional-params": "warn",
+    "no-link-to-tagname": "warn",
+    "no-log": "warn",
+    "no-negated-condition": "warn",
+    "no-nested-interactive": "warn",
+    "no-nested-landmark": "warn",
+    "no-nested-splattributes": "warn",
+    "no-obscure-array-access": "warn",
+    "no-obsolete-elements": "warn",
+    "no-outlet-outside-routes": "warn",
+    "no-partial": "warn",
+    "no-passed-in-event-handlers": "warn",
+    "no-pointer-down-event-binding": "warn",
+    "no-positional-data-test-selectors": "warn",
+    "no-positive-tabindex": "warn",
+    "no-potential-path-strings": "warn",
+    "no-quoteless-attributes": "warn",
+    "no-redundant-fn": "warn",
+    "no-redundant-role": "warn",
+    "no-route-action": "warn",
+    "no-scope-outside-table-headings": "warn",
+    "no-shadowed-elements": "warn",
+    "no-triple-curlies": "warn",
+    "no-unbalanced-curlies": "warn",
+    "no-unbound": "warn",
+    "no-unknown-arguments-for-builtin-components": "warn",
+    "no-unnecessary-component-helper": "warn",
+    "no-unsupported-role-attributes": "warn",
+    "no-unused-block-params": "warn",
+    "no-valueless-arguments": "warn",
+    "no-whitespace-for-layout": "warn",
+    "no-whitespace-within-word": "warn",
+    "no-with": "warn",
+    "no-yield-only": "warn",
+    "no-yield-to-default": "warn",
+    "require-aria-activedescendant-tabindex": "warn",
+    "require-button-type": "warn",
+    "require-context-role": "warn",
+    "require-has-block-helper": "warn",
+    "require-iframe-title": "warn",
+    "require-input-label": "warn",
+    "require-lang-attribute": "warn",
+    "require-mandatory-role-attributes": "warn",
+    "require-media-caption": "warn",
+    "require-presentational-children": "warn",
+    "require-valid-alt-text": "warn",
+    "require-valid-named-block-naming-format": "warn",
+    "simple-unless": "warn",
+    "splat-attributes-only": "warn",
+    "style-concatenation": "warn",
+    "table-groups": "warn",
```

Comment:
This is all needed? :fearful:

Reply:
AFAIK for everything to be a warning yes, not sure tho